src/preferences/dialog/dlgprefmixer.cpp

```diff
@@ -110,14 +114,40 @@ DlgPrefMixer::DlgPrefMixer(
     connect(SliderXFader,
             QOverload<int>::of(&QSlider::valueChanged),
             this,
-            &DlgPrefMixer::slotUpdateXFader);
-    connect(SliderXFader, &QSlider::sliderMoved, this, &DlgPrefMixer::slotUpdateXFader);
-    connect(SliderXFader, &QSlider::sliderReleased, this, &DlgPrefMixer::slotUpdateXFader);
-    connect(radioButtonAdditive, &QRadioButton::clicked, this, &DlgPrefMixer::slotUpdateXFader);
+            &DlgPrefMixer::slotXFaderWidgetsChanged);
+    connect(SliderXFader, &QSlider::sliderMoved, this, &DlgPrefMixer::slotXFaderWidgetsChanged);
+    connect(SliderXFader, &QSlider::sliderReleased, this, &DlgPrefMixer::slotXFaderWidgetsChanged);
+    connect(radioButtonAdditive,
+            &QRadioButton::clicked,
+            this,
+            &DlgPrefMixer::slotXFaderWidgetsChanged);
     connect(radioButtonConstantPower,
             &QRadioButton::clicked,
             this,
-            &DlgPrefMixer::slotUpdateXFader);
+            &DlgPrefMixer::slotXFaderWidgetsChanged);
+    connect(checkBoxReverse,
+            // TODO deprecated after Qt 6.9, replace with checkStateChanged(Qt::CheckState)
+            &QCheckBox::stateChanged,
+            this,
+            &DlgPrefMixer::slotXFaderReverseBoxToggled);
```

Comment:
lets add a `QT_VERSION_CHECK` for this now.

Reply:
Done, also for the other checkboxes.